.github/workflows/ci.yml

```diff
@@ -84,9 +84,7 @@ jobs:
             node: 20
     runs-on: ${{ matrix.os }}
     steps:
-      # Workaround for: https://github.com/bazel-contrib/bazel-lib/issues/968.
-      # TODO(devversion): Remove when Aspect lib issue is fixed.
-      - run: choco install gzip
+      - uses: Vampire/setup-wsl@v4
```

Comment:
This should use a commit SHA with the version in a comment afterwards. e.g., `@<some_sha> # v4`